Feature/tr 663 use review renderer on result page

btamas opened the pull request:
Related to https://oat-sa.atlassian.net/browse/TR-663

- Set renderer to `reviewRenderer` on Result page

Test process is described https://github.com/oat-sa/tao-item-runner-qti-fe/pull/157

jsconan reviewed (commented):
It works fine with respect to the review mode. However, it adds back the `Submit` button. There are at least 2 problems:
- the submit button must not be present for a review (not the same as for a preview)
- the submit action does not exist in this context, so that the UI cannot show any completion

![image](https://user-images.githubusercontent.com/1500098/109616370-5d13e480-7b35-11eb-92dc-8bcda67ff583.png)

btamas commented:
> It works fine with respect to the review mode. However, it adds back the `Submit` button. There are at least 2 problems:
> 
> * the submit button must not be present for a review (not the same as for a preview)
> * the submit action does not exist in this context, so that the UI cannot show any completion

Nice catch. The button appeared, because I removed `readOnly`, because it is not necessary with `reviewRenderer`. So, I added an extra condition to `submit` plugin here: https://github.com/oat-sa/extension-tao-testqti-previewer/pull/119

jsconan reviewed (approved):
Good job! It works well with the companion PRs.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
